### ISSUE TITLE
Allow tags to be assigned to ALB resource only

### ISSUE THREAD

**Issue by visit1985** [terraform-aws-modules/terraform-aws-alb#393]
I'd like to assign a `Name` tag to my ALB without having to overwrite the same on each listener/target-group etc.

https://github.com/terraform-aws-modules/terraform-aws-alb/blob/5121d715c42a2b63b0e02d41f675e5c5dd73ef50/main.tf#L68

Can we provide an additional variable like for security groups?

https://github.com/terraform-aws-modules/terraform-aws-alb/blob/5121d715c42a2b63b0e02d41f675e5c5dd73ef50/main.tf#L662

I can provide a pull request if required.

**Comment by bryantbiggs** [comment]
the name of the LB is set via `var.name` https://github.com/terraform-aws-modules/terraform-aws-alb/blob/5121d715c42a2b63b0e02d41f675e5c5dd73ef50/main.tf#L51

**Comment by visit1985** [comment]
Yes. Nevertheless I would like to create tags that apply to the LB only.
